eclair-core/src/main/scala/fr/acinq/eclair/payment/Bolt11Invoice.scala

```diff
@@ -89,10 +89,12 @@ case class Bolt11Invoice(prefix: String, amount_opt: Option[MilliSatoshi], creat
 
   lazy val relativeExpiry: FiniteDuration = FiniteDuration(relativeExpiry_opt.getOrElse(DEFAULT_EXPIRY_SECONDS), TimeUnit.SECONDS)
 
-  lazy val minFinalCltvExpiryDelta: Option[CltvExpiryDelta] = tags.collectFirst {
+  lazy val minFinalCltvExpiryDelta_opt: Option[CltvExpiryDelta] = tags.collectFirst {
```

Comment:
My first reaction was to delete `minFinalCltvExpiryDelta_opt`, do we really need to keep the granularity of knowing whether or not the field was actually defined in the invoice? It makes the API less foolproof, and that field was made mandatory almost two years ago (https://github.com/lightning/bolts/pull/785). WDYT? Let's at least add a comment:

```suggestion
  /**
   * @return the content of the 'min_final_cltv_expiry' field if present.
   *         Use [[minFinalCltvExpiryDelta]] if you need the applicable spec-defined value.
   */
  lazy val minFinalCltvExpiryDelta_opt: Option[CltvExpiryDelta] = tags.collectFirst {
```

(same for `relativeExpiry_opt`)

Reply:
The only reason I've kept it is so that the json output matches the invoice exactly. But it doesn't seem important so I've removed it.